### DISCUSSION TITLE
NextJS API returning 500

### DISCUSSION THREAD

**Discussion by yoga-teachedison** [vercel/community#1110] [Help] [answered]
I have written an API in NextJs, which hits another API from a different server and gets the response and process it. 

BROWSER  -------> Vercel Node --------> Django Server. 

This is working perfectly fine in local machine. But when deployed in Vercel, I get 500 error page with no other information. And it is random. Sometimes, the API is working fine and sometimes it fails. 

<img width="1418" alt="Screenshot 2022-11-28 at 8 43 54 PM" src="https://user-images.githubusercontent.com/75308636/204366747-95c31677-d344-4ea8-bebc-d764d1233068.png">

I would like to know if this error is because i am making an external API call in Next Node API.

**Comment by amyegan** [comment, marked as answer]
Sorry you're running into this error @yoga-teachedison. You should be able to see more details about the error in your project's [Function Logs](https://vercel.com/docs/concepts/deployments/logs#function-logs) or in your browser console or developer tools. You can find an [error code](https://vercel.com/docs/errors#) there, such as `FUNCTION_INVOCATION_FAILED` or `FUNCTION_INVOCATION_TIMEOUT`, to help with debugging.
